Providers/Modules/nx.psd1

```diff
@@ -0,0 +1,31 @@
+@{
+# Version number of this module.
+ModuleVersion = '<MODULE_VERSION>'
+
+# ID used to uniquely identify this module
+GUID = 'e2e610a5-8f47-4094-893c-539194be7501'
+
+# Author of this module
+Author = 'Microsoft Corporation'
+
+# Company or vendor of this module
+CompanyName = 'Microsoft Corporation'
+
+# Copyright statement for this module
+Copyright = '(c) 2015 Microsoft Corporation. All rights reserved.'
+
+# Description of the functionality provided by this module
+Description = 'Module with DSC MySQL Resources for Linux'
```

Comment:
This description looks like it needs to be updated. Check the GUID as well for uniqueness.